index.html

```diff
@@ -173,11 +184,13 @@ <h3>Professional art <span>printing Data</span></p>
         <button type="" class="project1-btn">See project</button>
```

Comment:
- [ ] Great job with adding the hover effect and the animation on hover 👏 However, kindly use the styling shown in Figma👍 
- [ ] Also please add the hover effect on the button in the contact section too 💯 

![image](https://user-images.githubusercontent.com/50721479/215274978-d972a057-1fa9-4443-b71b-4864e107e1f5.png)